src/components/problems/Card.jsx

```diff
@@ -1,6 +1,29 @@
 import { FaStar, FaRegStar } from "react-icons/fa";
+import { PiCopySimpleLight } from "react-icons/pi";
+import toast from "react-hot-toast";
 import { useState } from "react";
 
+const SolutionCode = ({ code }) => {
+  const handleCopy = () => {
+    navigator.clipboard.writeText(code);
+    console.log("copied");
```

Comment:
remove console log, if u need the function in multiple places put it in src/utils/Clipboard.js